src/StructuredLogger/TextUtilities.cs

```diff
@@ -451,18 +451,15 @@ public static string DisplayDuration(TimeSpan span, bool showZero = false)
             {
                 return showZero ? "0 ms" : "";
             }
-
-            if (span.TotalSeconds > 3600)
+            if (span.TotalHours >= 1)
             {
-                return span.ToString(@"h\:mm\:ss");
+                return span.ToString();

```

Comment:
Does this need to be stable? If so it might be better to specify the InvariantCulture.

Reply:
It's fine either way, it's for the user display only